ui/tests/integration/components/totp/page/totp-list-item-test.js

```diff
@@ -0,0 +1,46 @@
+import EmberObject from '@ember/object';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | totp-list-item', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    const item = EmberObject.create({
+      id: 'foo',
+      canRead: true,
+      canDelete: true,
+    });
```

Comment:
Using EmberObject is somewhat outdated and this is a good place to use mirage. Things will change as we move away from ember data, but that's the pattern we have now. you may be able to use this helper https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/secret-engine/secret-engine-helpers.js#L12